examples/example_app/docs/requirements.txt

```diff
@@ -3,3 +3,4 @@ mkdocs==1.6.0
 mkdocs-material==9.5.25
 mkdocstrings==0.24.1
 mkdocstrings-python==1.9.0
+zipp>=3.19.1 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
Our convention in the `docs/requirements.txt` (which is really only used by ReadTheDocs) is to pin to exact versions (`==` rather than `>=`) so as to guarantee reproducible documentation builds.